src/audio-view.js

```diff
@@ -0,0 +1,193 @@
+/*
+  Game event               Sound                      Logic
+
+  Enter alone mode         Alone_Mode/Pulse_amb_loop  FIXME
+  Handshake                Alone_Mode/Hand_Shake_01   changes.handshakes (Set of bool)
+
+Mole:
+  Panel activated          Game_01/G01_LED_XX
+  Active panel touched     Game_01/G01_Success_01
+  Non-active panel touched Game_01/G01_Negative_01
+
+Disk:
+
+Simon:
+  Panel activated during pattern animation
+                           Game_03/G03_LED_XX
+  Correct panel touched
+                           Game_03/G03_LED_XX
+  Wrong panel touched
+                           Game_03/G03_Negative_01
+  Won level (after short delay)
+                           Game_03/G03_Success_01
+  Won all levels (after short delay)
+                           Game_03/G03_Light_Show_01
+
+*/
+
+const _ = require('lodash');
+
+const SculptureStore = require('@anyware/game-logic/lib/sculpture-store');
+const GAMES = require('@anyware/game-logic/lib/constants/games');
+import {Sound, VCFSound} from './audio-api';
+
+export default class AudioView {
+  constructor(store, config, dispatcher) {
+    this.store = store;
+    this.config = config;
+  }
+
+  // Loads all sounds, calls callback([err]) when done
+  load(callback) {
+    // Maps logical sound identifiers to filenames. We'll load these sounds next.
+    this.sounds = {
+      alone: {
+        ambient: new VCFSound({url: 'sounds/Alone_Mode/Pulse_Amb_Loop.wav', fadeIn: 3}),
+        handshake: 'sounds/Alone_Mode/Hand_Shake_01.wav'
+      },
+      mole: {
+        success: 'sounds/Game_01/G01_Success_01.wav',
+        failure: 'sounds/Game_01/G01_Negative_01.wav',
+        panels: [0,1,2].map(stripId => _.range(10).map(panelId => `sounds/Game_01/G01_LED_${("0"+(stripId*10+panelId+1)).slice(-2)}.wav`))
```

Comment:
Sure. Since this should probably eventually end up in config anyway, I would populate it like this:

    // in config.js constructor:
    this.SOUNDS = {
      //...
      MOLE: {
        //...
        PANEL_ACTIVE: {
          // stripId
          '0': {
              // Note: SOUNDS_PATH.MOLE_GAME = "sounds/Game_01"
              // panelId
              '0': join(this.SOUNDS_PATH.MOLE_GAME, 'G01_LED_001.wav'),
              '1': join(this.SOUNDS_PATH.MOLE_GAME, 'G01_LED_002.wav'),
              // Example revised sound
              '2': join(this.SOUNDS_PATH.MOLE_GAME, 'G01_LED_003_rev.wav'),
              '3': join(this.SOUNDS_PATH.MOLE_GAME, 'G01_LED_004.wav'),
          }
        }
      }
      //...
    }
    // Note that this is just an example so you may come up with better names than I just did. :) 

There's a number of reasons I'd write the code like this and it has several benefits. 

All of these numbers that represent the strips and panels, though they look sequential, don't have to be. They're IDs with no defined order. We may have to change panel ID 2 to 7 one day because something went wrong. We shouldn't work on the assumption that they are sequential in our code. 

Another reason I'd do it like this is because while this may seem to be unnecessarily repetitive, it gives us a lot of freedom to configure the sounds individually however we like. Someone does the work to write this out once and we can adjust every sound whenever we want. In the code I have above, we're not limited by the seemingly sequential nature of the IDs that I was describing before. We can also have whatever filenames we want.

Imagine a scenario where we had two versions of a single panel's sound. What if we need to test that revision and want to swap a "_rev" prefix into the filename somewhere. It's worth it to have all of this laid out like this in the configuration class.

You can abstract out the name building as much as you'd like, even more than I did if it makes it easier. Just make sure that it's easy to configure and change.

This way of doing it also makes it easy to reason about and access from the code. You can simply use `[stripId][panelId]` on the relevant part of your configuration object. See how CONTROL_MAPPINGS is used in the disk view code.